lemmy_utils/src/email.rs

```diff
@@ -18,35 +20,63 @@ pub fn send_email(
   html: &str,
 ) -> Result<(), String> {
   let email_config = Settings::get().email.ok_or("no_email_setup")?;
+  let domain = &Settings::get().hostname;
 
-  let email = Email::builder()
-    .to((to_email, to_username))
-    .from(email_config.smtp_from_address.to_owned())
+  let email = Message::builder()
+    .from(
+      email_config
+        .smtp_from_address
+        .parse()
+        .expect("from address isn't valid"),
```

Comment:
Maybe put `email from address isnt valid`? Depends how this would end up in the log.